src/Pool.sol

```diff
@@ -322,6 +345,25 @@ abstract contract Pool {
         }
     }
 
+    function _collectCycles(address receiverAddr, uint64 count)
```

Comment:
I think we should agree what a collect means

-collect
  - flushCycle
  - drip

If thats the case, we should call this method flushCycleInternal :)

Reply:
Good point! Fixed